libs/wallet/src/order-hooks/use-order-edit.tsx

```diff
@@ -0,0 +1,82 @@
+import { useCallback, useEffect, useState } from 'react';
+import { determineId } from '@vegaprotocol/react-helpers';
+
+import { useVegaTransaction } from '../use-vega-transaction';
+import { useVegaWallet } from '../use-vega-wallet';
+import { useSubscription } from '@apollo/client';
+import type {
+  OrderEvent,
+  OrderEventVariables,
+  OrderEvent_busEvents_event_Order,
+} from './__generated__/OrderEvent';
+import { ORDER_EVENT_SUB } from './order-event-query';
+
+export const useOrderEdit = () => {
+  const { keypair } = useVegaWallet();
+  const { send, transaction, reset: resetTransaction } = useVegaTransaction();
+  const [finalizedOrder, setFinalizedOrder] =
```

Comment:
suggestion(non blocking):
```suggestion
  const [editedOrder, setEditedOrder] =
```

Reply:
this needs to come in another PR -  i will remove it for now